packages/app-next/app-config.yaml

```diff
@@ -21,7 +25,25 @@ app:
         config:
           height: 300
     - entity-card:azure-devops/readme
+    - entity-card:api-docs/has-apis:
+        config:
+          filter: kind:component
```

Comment:
@freben Is there a way of only show this if the component has apis relations?

Reply:
Only want right now would be to make that the built-in filter function.

Reply:
Got it, so adopters can override the extension and pass a filter function, right?

Reply:
Rather, if its default implementation was to return a `catalogExtensionData.entityFilterFunction` in its output that does that exact check.

The tricky thing is, what would this particular line in the config mean? Normally the expectation would be that if you added a filter expression in config, it would _replace_ the `entityFilterFunction` rather than amend it, which is a problem - because there's no way of saying `has-relation:hasApi` in the filter strings as of yet.

We could add that as a feature to the filter strings if we want. But on the other hand, maybe it's going to be very uncommon to write a custom filter string? Very few things have api relations in the first place, and how likely is it that you want to only show this card for a subset of those? Maybe it's a non-issue in practice.

So for now, I'd say: make the default implementation return the correct filter function, and remove these filter overrides in the example because they will override that experience potentially making it worse.

Reply:
Thank you for explaining 🙏🏻 . I removed the configs and defined default filters. Could you take a look again?